client/packages/common/src/ui/components/inputs/Select/Select.tsx

```diff
@@ -17,33 +19,69 @@ const defaultRenderOption = (option: Option) => (
 );
 
 export const Select: FC<SelectProps> = React.forwardRef(
-  ({ options, renderOption, sx, InputProps, ...props }, ref) => (
-    <TextField
-      ref={ref}
-      sx={{
-        '& .MuiInput-underline:before': { borderBottomWidth: 0 },
-        '& .MuiInput-input': { color: theme => theme.palette.gray.dark },
-        ...sx,
-      }}
-      select
-      variant="standard"
-      size="small"
-      InputProps={{
-        disableUnderline: true,
-        ...InputProps,
-        sx: {
-          backgroundColor: theme =>
-            props.disabled
-              ? theme.palette.background.toolbar
-              : theme.palette.background.menu,
-          borderRadius: '8px',
-          padding: '4px 8px',
-          ...InputProps?.sx,
-        },
-      }}
-      {...props}
-    >
-      {options.map(renderOption || defaultRenderOption)}
-    </TextField>
-  )
+  (
+    {
+      options,
+      renderOption,
+      sx,
+      InputProps,
+      clearable = false,
+      value,
+      onChange = () => {},
+      ...props
+    },
+    ref
+  ) => {
+    const SelectProps =
+      clearable && value
+        ? {
+            IconComponent: () => (
+              // Element to clear current selection.
+              <CloseIcon
+                onClick={e =>
+                  onChange({ ...e, target: { value: undefined } } as any)
+                }
+                style={{
+                  fontSize: '1em',
+                  cursor: 'pointer',
+                  color: 'rgba(0, 0, 0, 0.54);',
```

Comment:
This colour just copy-pasted from icon "pointer" icon that is there initially. Any way to refer to a system/Mui variable here instead?

Reply:
Tried copying the classNames from the default icon, but that didn't help.

Reply:
It's not perfect but I would change to `sx` rather than `style` and use `color: 'gray.dark'`
Really trying hard to keep hard coded colours out of the codebase, and always use the theme.